gadgets/src/less_than.rs

```diff
@@ -89,7 +94,27 @@ impl<F: Field, const N_BYTES: usize> LtChip<F, N_BYTES> {
                 .map(move |poly| q_enable.clone() * poly)
         });
 
-        LtConfig { lt, diff, range }
+        for cell_columns in diff.chunks(2) {
+            meta.lookup("range check for u16", |meta| {
+                let q_enable = q_enable.clone()(meta);
+                let cell_expr = if cell_columns.len() == 2 {
+                    meta.query_advice(cell_columns[0], Rotation::cur())
```

Comment:
This does not work as a range check because the columns are individually not constrained.

It’s good to use a u16 table though. But each column should be u16 (with twice fewer columns).

When N_BYTES is odd, it’s fine to round up to the next even number. Make sure that the `range` constant also uses the rounded up number.

Reply:
hmmm, you are right. that needs a lot of changes, since we can't div and ceil in const generics, this param need to be changed to N_2BYTES

Reply:
U8 is also not too bad because actual instances use only 1, 2, or 3 bytes ranges.